tools/packaging/packaging.go

```diff
@@ -141,6 +141,15 @@ func CreateLinuxPackages(osqueryVersion, hostname, secret string, insecure, inse
 		}
 	}
 
+	rootPEMPath := ""
```

Comment:
nit: I would prefer to use the zero value then initializing an empty string like this in Go code.